examples/next/faustwp-getting-started/next.config.js

```diff
@@ -15,4 +18,4 @@ module.exports = withFaust({
     locales: ['en'],
     defaultLocale: 'en',
   },
-});
+}));
```

Comment:
We should include this code programatically so the end user doesn't have this setup on their site. Something like:

```bash
echo "const withBundleAnalyzer = require('@next/bundle-analyzer')({enabled: true})\nmodule.exports = withBundleAnalyzer(module.exports)" >> next.config.js
```

This will add the following lines to the `next.config.js` file programatically:

```js
const withBundleAnalyzer = require('@next/bundle-analyzer')({ enabled: true });
module.exports = withBundleAnalyzer(module.exports);
```

So the full file would look like:

```js
const { withFaust, getWpHostname } = require('@faustwp/core');

/**
 * @type {import('next').NextConfig}
 **/
module.exports = withFaust({
  reactStrictMode: true,
  sassOptions: {
    includePaths: ['node_modules'],
  },
  images: {
    domains: [getWpHostname()],
  },
  i18n: {
    locales: ['en'],
    defaultLocale: 'en',
  },
});
const withBundleAnalyzer = require('@next/bundle-analyzer')({ enabled: true });
module.exports = withBundleAnalyzer(module.exports);
```